lib/maze/option/parser.rb

```diff
@@ -32,10 +32,15 @@ def parse(args)
                 default: true
 
             opt Option::ENABLE_BUGSNAG,
-                'Enables reporting to Bugsnag on scenario failure (Require MAZE_BUGSNAG_API_KEY)',
+                'Enables reporting to Bugsnag on scenario failure (requires MAZE_BUGSNAG_API_KEY)',
                 type: :boolean,
                 default: true
 
+            opt Option::REPEATER,
+                'Enables forwarding of all received POST requests to Bugsnag using the API key held in MAZE_REPEATER_API_KEY',
+                type: :boolean,
+                default: false
```

Comment:
Have you considered making the API Key the option and enabling/disabling it based on the presence of the key?

i.e. instead of:

```
$ MAZE_REPEATER_API_KEY=abc bundle exec maze-runner --repeater
```

do:

```
$ bundle exec maze-runner --repeater-api-key abc
```

There might be a reason to use an environment variable, but I think it's a little nicer not to have to specify two things. Though maybe it's also useful to always provide an API key, but allow controlling whether to actually use it separately?